test_ocp_versions.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -euo pipefail
+
+SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+
+OPENSHIFT_SNO_IMAGE="registry.svc.ci.openshift.org/sno-dev/openshift-bip:0.5.0"
+
+${SCRIPT_DIR}/tools/handle_ocp_versions.py --src ./default_ocp_versions.json --ocp-override registry.svc.ci.openshift.org/sno-dev/openshift-bip:0.5.0 --name-override single-node-4.8-alpha --version-override 4.8 --skip-verify
```

Comment:
1. Why it is added to this level? It means that staging/integration won't have this version. In case it is needed, I think we should go back to maintain multiple JSONs.
2. Why is the verification being skipped? This image isn't a 4.8 one?

Reply:
1. Exactly because we *don't* want it in staging / production
2. Because this script is being called every time someone runs make, and verification takes a couple of seconds, it would be very annoying. This image is a 4.8 one, but I put it there, so I know it, and I don't need to check if every time someone runs make.